statistics-client-ingest/src/main/java/no/difi/statistics/client/HttpHelper.java

```diff
@@ -0,0 +1,72 @@
+package no.difi.statistics.client;
+
+import no.difi.statistics.client.exception.DifiStatisticsException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.ProtocolException;
+import java.net.URL;
+
+public class HttpHelper {
+
+    private static final String REQUEST_METHOD_POST = "POST";
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    protected OutputStream getOutputStream(HttpURLConnection conn) throws DifiStatisticsException {
+        OutputStream outputStream = null;
```

Comment:
Ser ikke at denne hjelpemetoden gir noen verdi. HttpURLConnection.getOutputStream gir aldri en null-verdi, og unntaksoversettelsen er overflødig (for API-bruker spiller det ingen rolle om han får en IOException eller en DifiStatisticsException).